catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/EventProcessorImpl.java

```diff
@@ -435,8 +435,4 @@ public DeleteResponse process(DeleteResponse deleteResponse) throws PluginExecut
         return deleteResponse;
     }
 
-    public static enum DateType {
```

Comment:
One way to work around this without having to switch from `enum` to `String` would be to add a constructor to `DateType` that takes in the expected name (e.g., `metacard.modified`) and have a method on the `enum` to return it, e.g., `getAttributeName()` or something.